docs/vNext/Fabric_Configuration_New.md

```diff
@@ -0,0 +1,1115 @@
+
+### Using the Operational capabilities of the Adapters
+The ability to create channels and perform install/instantiate of contracts (chaincode) remains in the new 1.x sdk based adapters, and nothing equivalent exists for the 2.x sdk based adapters. This capability remains mainly to support the caliper integration tests and the use of these mechanisms to setup your channels and contracts is highly discouraged. This capability is considered legacy and will not be maintained. You should look into alternative methods for setting up your network prior to running caliper to benchmark it.
+
+## Runtime settings
+
+### Common settings
+
+Some runtime properties of the adapter can be set through Caliper's [runtime configuration mechanism](./Runtime_Configuration.md). For the available settings, see the `caliper.fabric` section of the [default configuration file](https://github.com/hyperledger/caliper/blob/master/packages/caliper-core/lib/common/config/default.yaml) and its embedded documentation.
+
+The above settings are processed when starting Caliper. Modifying them during testing will have no effect. However, you can override the default values _before Caliper starts_ from the usual configuration sources.
+
+> __Note:__ An object hierarchy in a configuration file generates a setting entry for every leaf property. Consider the following configuration file:
+> ```yaml
+> caliper:
+>   fabric:
+>     gateway:
+>       enabled: true
+> ```
+> After naming the [project settings](./Runtime_Configuration.md#project-level) file `caliper.yaml` and placing it in the root of your workspace directory, it will override the following two setting keys with the following values:
+> * Setting `caliper-fabric-gateway-enabled` is set to `true`
+>
+> __The other settings remain unchanged.__
+
+### Skip channel creation
+
+Additionally, the adapter provides a dynamic setting (family) for skipping the creation of a channel. The setting key format is `caliper-fabric-skipcreatechannel-<channel_name>`. Substitute the name of the channel you want to skip creating into `<channel_name>`, for example:
+
+```console
+user@ubuntu:~/caliper-benchmarks$ export CALIPER_FABRIC_SKIPCREATECHANNEL_MYCHANNEL=true
+```
+
+> __Note:__ This settings is intended for easily skipping the creation of a channel that is specified in the network configuration file as "not created". However, if you know that the channel always will be created during benchmarking, then it is recommended to denote this explicitly in the network configuration file.
+
+Naturally, you can specify the above setting multiple ways (e.g., command line argument, configuration file entry).
+
+## The connector API
+
+The [workload modules](./Workload_Module.md) interact with the adapter at two phases of the tests: during the initialization of the user module (in the `initializeWorkloadModule` callback), and when submitting invoke or query transactions (in the `submitTransaction` callback).
```

Comment:
there are 3 phases - there is an optional cleanup stage too

Reply:
Thanks Nick, added a bit about cleanup